easybuild/easyconfigs/l/libgd/libgd-2.2.4-GCCcore-6.4.0.eb

```diff
@@ -19,9 +19,9 @@ builddependencies = [
 ]
 
 dependencies = [
-    ('fontconfig', '2.12.4'),
-    ('libjpeg-turbo', '1.5.2'),
-    ('libpng', '1.6.32'),
+    ('fontconfig', '2.12.6'),
+    ('libjpeg-turbo', '1.5.3'),
+    ('libpng', '1.6.34'),
```

Comment:
@reedts Please don't bump dependency versions in existing easyconfig, you'll introduce conflicts in other places...

Reply:
I'll take it out

Reply:
@reedts You want to add a new easyconfig for `libgd` 2.2.5`, not change the existing one for `2.2.4` in place...

Reply:
Ah I found an error in my libgd-2.2.5 easyconfig. The version there was still 2.2.4. Seems like it messed things up...